CMakeLists.txt

```diff
@@ -615,4 +627,29 @@ else ()
   endif ()
 endif (IS_64BIT_BUILD EQUAL 1)
 
+
+# The prebuilt IDL headers aren't suitable for 32 bit builds, attempt to use the ones 
+# generated when building corguids by including them before the prebuilt ones
+if (WIN32 AND CLR_CMAKE_PLATFORM_ARCH_I386)
```

Comment:
Thanks, I reverted all IDL & prebuilt header related changes.

It builds fine on my machine but the CI server seems to have gone on holiday :tropical_drink: